src/Microsoft.Diagnostics.DebugServices.Implementation/ServiceManager.cs

```diff
@@ -334,8 +341,8 @@ private Assembly UseAssemblyLoadContext(string extensionPath)
             Assembly assembly = extension.LoadFromAssemblyPath(extensionPath);
             if (assembly is not null)
             {
-                // This list is just to keep the load context alive
-                _extensions.Add(extension);
+                // This list to track the assemblies loaded and to keep the load context alive (ExtensionLoadContext)
+                _extensions.Add((assembly, extension));
```

Comment:
Isn't the assembly enough to keep the context alive?

Reply:
I don't want to take the chance.  The original code referenced the AssemblyLoadContext so I want to keep it that way.